src/aat/resources/features/F-037/S-577.td.json

```diff
@@ -0,0 +1,156 @@
+          "document_filename" : "Screenshot 2019-11-13 at 9.52.02 am.png",
+          "document_binary_url" : "http://dm-store:8080/documents/68c9f85f-1d0c-4d02-b082-502a87c1ac6c/binary"
+        },
+        "FixedListField" : "VALUE3",
+        "AddressUKField" : {
+          "County" : null,
+          "Country" : null,
+          "PostCode" : null,
+          "PostTown" : null,
+          "AddressLine1" : null,
+          "AddressLine2" : null,
+          "AddressLine3" : null
+        },
+        "DocumentField6" : {
+          "document_url" : "http://dm-store:8080/documents/76976634-374f-472b-a819-a3a938b4cc83",
+          "document_filename" : "Screenshot 2019-11-13 at 11.25.45 am.png",
+          "document_binary_url" : "http://dm-store:8080/documents/76976634-374f-472b-a819-a3a938b4cc83/binary"
+        },
+        "DocumentField5" : {
+          "document_url" : "http://dm-store:8080/documents/f96735a6-8764-4778-978c-e7612a610a94",
+          "document_filename" : "Screenshot 2019-11-18 at 10.20.24 am.png",
+          "document_binary_url" : "http://dm-store:8080/documents/f96735a6-8764-4778-978c-e7612a610a94/binary"
+        },
+        "DocumentField2" : {
+          "document_url" : "http://dm-store:8080/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_binary_url" : "http://dm-store:8080/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary",
+          "document_filename" : "Elastic Search test Case.png --> updated by Solicitor 2"
+        },
+        "DocumentField1" : {
+          "document_url" : "http://dm-store:8080/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4",
+          "document_filename" : "Screenshot 2019-11-13 at 3.24.26 pm.png",
+          "document_binary_url" : "http://dm-store:8080/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4/binary"
+        },
+        "ComplexField" : {
+          "ComplexTextField" : null,
+          "ComplexNestedField" : {
+            "NestedNumberField" : null,
+            "NestedCollectionTextField" : [ ]
+          },
+          "ComplexFixedListField" : null
+        },
+        "FixedRadioListField" : "RADIOVALUE2",
+        "DateTimeField" : null,
+        "PhoneUKField" : null,
+        "DocumentField7" : {
+          "document_url" : "http://dm-store:8080/documents/5c14f8ef-887b-41e0-ad01-23959c082311",
+          "document_filename" : "Screenshot 2019-12-24 at 3.41.04 pm.png",
+          "document_binary_url" : "http://dm-store:8080/documents/5c14f8ef-887b-41e0-ad01-23959c082311/binary"
+        },
+        "NumberField" : "111111222222",
+        "MultiSelectListField" : [ "OPTION5", "OPTION4", "OPTION3" ],
+        "YesOrNoField" : "Yes",
+        "EmailField" : "testemail@gmail.com",
+        "TextField" : "sometextfield",
+        "DateField" : "2222-01-02",
+        "TextAreaField" : "textarea",
+        "CollectionField" : [ ]
+      },
+      "data_classification" : {
```

Comment:
could create a `Befta_Default_Full_Case_Classification_Data.td.json` file and extend it here

Reply:
Yes, it should be taken out and reused. Done.